.travis.yml

```diff
@@ -5,22 +5,27 @@
 # we will use conda to give us a much faster setup time.
 
 
-language: python
-python:
-  - 2.7
-  - 3.6
-sudo: false
+language: minimal
+dist: xenial
```

Comment:
@pp-mo I don't see the harm in being explicit here. I think it's good to define and control the Travis CI build environment, rather than have it change under our feet - particularly when it comes to testing.

Reply:
Ok, get it 👍